tools/main.py

```diff
@@ -62,8 +63,12 @@ def main():
     log_name = 'test.log' if cfg.test.evaluate else 'train.log'
     log_name += time.strftime('-%Y-%m-%d-%H-%M-%S')
     sys.stdout = Logger(osp.join(cfg.data.save_dir, log_name))
+    branch_name = subprocess.check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD']).decode('ascii').strip()
```

Comment:
I'd move that stuff to a standalone function. Also the directory from which the command is called is important. If we call main.py not from the d-o-r repo, this command would fail or fetch a wrong commit. I propose to rely on source code file path and use `cwd` parameter.

Reply:
done